### PR TITLE
Initial commit of vm-factory

### PR DESCRIPTION
**Pull request by agutta**

### CODE REVIEW

File: blueprints/factories/vm-factory/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This factory is based on the [compute-vm module](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/compute-vm)
+
+You can create as many files as you like, the code will loop through it and create the required variables in order to execute everything accordingly.
+
+## Example
+
+### Terraform code
+
+```hcl
```
Comment (ludoo) [inline blueprints/factories/vm-factory/README.md:15]: Anita, I'm struggling to understand how this example is a factory...

**Comment by juliocc** [comment]
@agutta is this ready to be reviewed?

File: blueprints/factories/vm-factory/README.md (hunk 1 above)
Comment (ludoo) [reply blueprints/factories/vm-factory/README.md:15]: I'm not seeing the yaml referenced in the example, and I see for_each at the module level...

Comment (ludoo) [reply blueprints/factories/vm-factory/README.md:89]: A factory should take a data path, and internally parse files and manage resources. Look to the net-vpc or folder factories for examples. We also don't much like using modules inside factories, that creates too many layers of abstraction.

Comment (ludoo) [reply blueprints/factories/vm-factory/README.md:89]: I don't know if an example that does a for each on the vm module is complex enough to put in the repo... let's close this and discuss live?

**Pull request closed without merging**